### ISSUE TITLE
Little more Netplay weirdness for you guys (traversal prevents overlay loading)

### ISSUE THREAD

**Issue by klepp0906** [libretro/RetroArch#9426]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

Sorry about the title.  Its late'ish and honestly, no idea.  Feel free to edit it to something more accurate if you can come up with something.

So wrapping up working on netplay for tonight, I found that if you enable NAT traversal, if you have an overlay (border some call them) set to load with your config override, it wont load.  This is dependent on Use Relay Server being disabled and you starting a game as netplay host.

AKA:
-Relay Server enable & NAT Traversal enable = border WILL load.  
-Relay Server disable & NAT Traversal enable = border WONT display (yet popup indicates override loaded, and shader preset still loads)
-NAT Traversal disable = border WILL load. 

This is 
### Expected behavior

The aforementioned settings to have no effect on overlays/borders whatsoever.  I thought perhaps it was forced due to some kind of syncing or latency prevention thing, so asked in the discord - but sounded like its not supposed to happen so i figured id come here and do my part by harassing you guys. 

### Actual behavior

In the aforementioned scenarios I end up with a big black screen with a curved video game (from the shader) in the center instead of surrounded by an overlay.

### Steps to reproduce the bug

1. Create a content directory override that includes the addition of an overlay/border.
2. Toggle on NAT traversal ensuring Relay Server is disabled.
3. Start as netplay host.
4. load a game (in my case SF2 Turbo/SNES on mercury-accuracy)
5. note your overlay isnt showing.

### Bisect Results

Just began with netplay tonight, so I cant give ya anything to this end :(

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.78 97d91a895c

### Environment information

- OS: Windows 10

**Comment by klepp0906** [comment]
So, as I was posting this - I was talking with wambulance in the discord.   Two things of note.

1) the overlay does pop up, about 10 seconds later after "port mapping failed" notification (since I dont have upnp).

2) he said I should submit an issue report anyways, as this is happening due to the order in which traversal is loaded (a task) which should happen after override loading, not before.

So basically my entire issue report is #2 above I guess.

**Comment by LibretroAdmin** [comment]
@Cthulhu-throwaway Is this issue still relevant or can it be closed?
